go/testing/action.go

```diff
@@ -18,6 +20,8 @@ type Driver interface {
 	Advance(TestTime) error
 }
 
+var ErrAdvanceDisabled = errors.New("advance disabled")
```

Comment:
nit: add comment for exported types do you have a Go linter set up?).

Reply:
Sorry, I'm used to having linting part of of CI and VS Code seems to be grumpy using staticheck with all the CGooeyness. I tried Golint and it seems happier.